### PR TITLE
Redirect to Documents

### PR DESCRIPTION
**Pull request by davidwessman**
- Uses remote url instead of downloading file and showing 
(they do not need to be protected and it takes a lot of memory to process them)
- Remove capistrano files (copied from other app)
- Update `bin/setup`

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -12,13 +12,8 @@ Dir.chdir APP_ROOT do
...
   system "bundle check || bundle install"
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?("config/database.yml")
-  #   system "cp config/database.yml.sample config/database.yml"
-  # end
-
   puts "\n== Preparing database =="
-  system "bin/rake db:setup"
+  system "bin/rake db:setup permissions:load db:populate_test"
```
Comment (navro) [inline bin/setup]: Does heroku run this file? Or is it only for `development`?

Comment (davidwessman) [inline bin/setup]: Just for new developers, so you run `bin/setup` and then you can start the server.

Comment (navro) [inline bin/setup]: Great!

**Comment by navro** [comment]
+1
👍

**Pull request merged**